catalog/core/catalog-core-api-impl/src/test/java/ddf/catalog/impl/MockMemoryProvider.java

```diff
@@ -91,6 +91,11 @@
 
     private String sourceId = "mockMemoryProvider";
 
+    public MockMemoryProvider(String id, boolean isAvailable) {
+        this(id, "", "", "", new HashSet<>(), isAvailable, new Date());
+        sourceId = id;
```

Comment:
It was originally hard coded and not changeable. I made it changeable with the new constructor. Moved setting of sourceId to the constructor.